http/src/ratelimiting/mod.rs

```diff
@@ -42,7 +42,7 @@ impl GlobalLockPair {
     }
 }
 
-#[derive(Debug, Default)]
+#[derive(Clone, Debug, Default)]
```

Comment:
Is this change related to this PR?

Reply:
Yes it is required for RatelimiterBuilder to be Clone which is required for ClientBuilder to be Clone